### ISSUE TITLE
setting RUSTFLAGS for linux

### ISSUE THREAD

**Issue by ritchie46** [conda-forge/polars-feedstock#7]
### Comment:

For linux we build polars with the following RUSTFLAGS as environment var:

```bash
export RUSTFLAGS='-C target-feature=+fxsr,+sse,+sse2,+sse3,+ssse3,+sse4.1,+sse4.2,+popcnt,+avx,+fma'

```

Where should I put these. I see a `build.sh`, but I think that will also be used by os-x. 

Is it best to set it in the `run_docker_build.sh`?

**Comment by ritchie46** [comment]
We don't have runtime detectable simd in polars. We could define a minimal set. This is what I have done in the polars CI as well. 
The current set is the one where we did not get an issue that it was not working for them. ;)

I could drop the `avx` feauture (note that it is not avx512). That would already be far more than the rustc defaults.

**Comment by ritchie46** [comment]
fixed by #11
